### ISSUE TITLE
Upgrade jest-axe to ^5.0.0

### ISSUE THREAD

**Issue by halocline** [grommet/grommet#5502]
### Expected Behavior

Project should stay current with latest accessibility rules and check for compliance within testing suite. 

### Actual Behavior

Unable to upgrade to `"jest-axe": "^5.0.1"` (which relies on `axe-core 4.20`) because upgrade results in the following new test failures.

Blocked by:
- [x] https://github.com/grommet/grommet/issues/5501
- [x] https://github.com/grommet/grommet/issues/5500

### Steps to Reproduce

- Modify `package.json` to use `"jest-axe": "^5.0.1"`
- `yarn install`
- `yarn test`

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
